zerver/actions/invites.py

```diff
@@ -230,36 +231,36 @@ def do_invite_users(
                 sent_invitations=False,
             )
 
-    good_emails: set[str] = set()
+    good_invitees: set[Invitee] = set()
     errors: list[tuple[str, str, bool]] = []
     validate_email_allowed_in_realm = get_realm_email_validator(realm)
-    for email in invitee_emails:
-        if email == "":
-            continue
+    for invitee in invitees:
+        if invitee.email == "":
```

Comment:
I don't think this check is needed, as there is `if email` check in `get_invitees_set` and there is only one call to `do_invite_users` except tests.